onyo/tests/test_usecases.py

```diff
@@ -0,0 +1,205 @@
+import subprocess
+import pytest
+from pathlib import Path
+
+from onyo.lib.onyo import OnyoRepo
+
+
+prefilled_assets = [["warehouse/monitor_dell_PH123.86JZho",
+                     "display: '22.0'\nfzj_inventory: '45FZ18'\n"],
+                    ["warehouse/monitor_dell_NoIdea.S0M3",
+                     "display: '27.0'\n"],
+                    ["warehouse/laptop_apple_macbook.oiw629",
+                     "RAM: '8GB'\ndisplay: '13.3'\nUSB_A: '2'\nfzj_inventory: '28FZ34'\nbuild-date: '20160501'\n"],
```

Comment:
I notice that you quote all numbers (e.g. `display: '13.3'` and `USB_A: '2'`) even though YAML does know integers and floats.

The idea is around that we want to be able to sum up numbers in queries (obviously more relevant for storage then display sizes, but still). So I think these assets would come closer to real use cases when they would contain actual numbers :)

Reply:
Yes, I just took that from what I saw in other tests and the demo repo. I believe this is not yet consistently handled.

> The idea is around

Is there an issue for this idea?

Reply:
Nope, but it is probably a requirement for asset folders. I will create one!

Reply:
https://github.com/psyinfra/onyo/issues/363